api/pyproject.toml

```diff
@@ -42,9 +42,9 @@ scikit-learn = "^1.1.3"
 httpx = "^0.27.0"
 orjson = "^3.9.0"
 greenlet = "^3.0.0"
-sentry-sdk = {extras = ["fastapi"], version = "^2.0.1"}
+sentry-sdk = { extras = ["fastapi"], version = "^2.0.1" }
 numba = "^0.59.1"
-cffdrs = {version = "^0.1.1", source = "psu"}
+cffdrs = { git = "https://github.com/cffdrs/cffdrs_py.git" }
```

Comment:
Nice! Can we tie it to a specific commit like we do in the frontend?

Reply:
Yep I believe we can